modules/virt-about-upgrading-virt.adoc

```diff
@@ -28,14 +28,14 @@ virtual machine.
 +
 [NOTE]
 ====
-Each virtual machine has a `virt-launcher` Pod that runs the virtual machine
-instance. The `virt-launcher` Pod runs an instance of `libvirt`, which is
+Each virtual machine has a `virt-launcher` pod that runs the virtual machine
+instance. The `virt-launcher` pod runs an instance of `libvirt`, which is
 used to manage the virtual machine process.
 ====
 
 * Upgrading does not interrupt network connections.
 
-* DataVolumes and their associated PersistentVolumeClaims are preserved during
+* data volumes and their associated persistent volume claims are preserved during
```

Comment:
If there's a bullet point `* ` preceding the term, we should capitalize the first letter.

Reply:
:clinking_glasses: